_includes/v21.1/orchestration/start-cockroachdb-helm-insecure.md

```diff
@@ -43,13 +43,31 @@ The CockroachDB Helm chart is undergoing maintenance for compatibility with Kube
         For example, if you are allocating 8Gi of `memory` to each CockroachDB node, allocate 2Gi to `cache` and 2Gi to `max-sql-memory`.
         {{site.data.alerts.end}}
 
-    2. You may want to modify `storage.persistentVolume.size` and `storage.persistentVolume.storageClass` for your use case. This chart defaults to 100Gi of disk space per pod. For more details on customizing disks for performance, see [these instructions](kubernetes-performance.html#disk-type).
+    1. For an insecure deployment, set `tls.enabled` to `false`. For clarity, this example includes the example configuration from the previous steps.
+
+        {% include_cached copy-clipboard.html %}
+        ~~~
+        statefulset:
+          resources:
+            limits:
+              memory: "8Gi"
+            requests:
+              memory: "8Gi"
+        conf:
+          cache: "2Gi"
+          max-sql-memory: "2Gi"
+        tls:
+          enabled: false
+        ~~~
+
+    1. You may want to modify `storage.persistentVolume.size` and `storage.persistentVolume.storageClass` for your use case. This chart defaults to 100Gi of disk space per pod. For more details on customizing disks for performance, see [these instructions](kubernetes-performance.html#disk-type).
 
         {{site.data.alerts.callout_info}}
         If necessary, you can [expand disk size](operate-cockroachdb-kubernetes.html#expand-disk-size) after the cluster is live.
```

Comment:
Not technically part of your PR, but might be worth changing this link to `configure-cockroachdb-kubernetes?filters=helm#expand-disk-size` (the new location on `stable`) while we're here